CoroutineTemplate/data/src/main/java/co/nimblehq/coroutine/data/response/Response.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.coroutine.data.response
+
+import co.nimblehq.coroutine.domain.model.Model
+import com.squareup.moshi.Json
+
+data class Response(
+    @Json(name = "id") val id: Int?
+)
+
+private fun Response.toModel(): Model {
+    return Model(
+        id = this.id
+    )
+}
```

Comment:
What do you think about updating this function a bit for shorter code? 🤔 
```suggestion
private fun Response.toModel(): Model = Model(id = this.id)
```